client/client.go

```diff
@@ -0,0 +1,32 @@
+package client
```

Comment:
Missing package level documentation... Also `client` is not a great package name...

How about `tunnelclient`? or.... Think about how this will look when important in an other project... `client` is a super generic... also you shouldn't use packages names that users might want to use as variable names...